qdyn/problem_class.f90

```diff
@@ -155,6 +162,10 @@ module problem_class
                                   P_glob(:) => null(), T_glob(:) => null()
     double precision, pointer ::  tau_max_glob(:) => null(), t_rup_glob(:) => null(), &
                                   v_max_glob(:) => null(), t_vmax_glob(:) => null()
+    ! CRP: quantities related to vmax_fault
+    ! integer, pointer :: ivmax_fault_glob(:) => null()
+    ! double precision, pointer ::  vmax_fault_glob(:) => null()
```

Comment:
Why are these commented out?

Reply:
because on a first attempt I was trying to retrieve the values per processor, but I ended up doing it after gathering all the values, so at the end I don't need those variables. I removed the commented lines